pages/04.HDM Doc 2.2/08.vcs/60.Undeploy/default.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Undelpoy Steps'
+title: 'Undeploy Steps'
```

Comment:
Is this change only required in pages & not in macros?

Reply:
Yes this is only at one place in vcs pages.

Reply:
Ok, fine